lightbluetent/admin.py

```diff
@@ -314,10 +314,10 @@ def admin(uid):
                         society.socials.append(social_data)
                         flag_modified(society, "socials")
 
-            society.description = values["description"]
-            society.short_description = values["short_description"]
-            society.welcome_text = values["welcome_text"]
-            society.banner_text = values["banner_text"]
+            society.description = values["description"] if values["description"] != "" else None
+            society.short_description = values["short_description"] if values["short_description"] != "" else None
+            society.welcome_text = values["welcome_text"] if values["welcome_text"] != "" else None
+            society.banner_text = values["banner_text"] if values["banner_text"] != "" else None
```

Comment:
In the future, it might be convenient to define a util like `nif` that takes an arg and does this so all we have to do is call nif(values["banner_text"]) and it looks cleaner. but that's just personal style -- looks good